src/RegistrationManager.ts

```diff
@@ -208,10 +208,13 @@ export class RegistrationManager {
         return transactionHashes;
     }
 
-    async getRelayData(): Promise<RelayData> {
-        const relayData: RelayData[] =
-            await this.contractInteractor.getRelayData([this.managerAddress]);
-        if (relayData.length > 0) {
+    async getRelayData(): Promise<RelayManagerData> {
+        const relayData: RelayManagerData[] =
+            await this.contractInteractor.getRelayInfo([this.managerAddress]);
+        if (relayData.length > 1) {
+            throw new Error('More than one relay manager found for ' + this.managerAddress)
+        }
```

Comment:
it _shouldn't_ happen, but if it does, it means that the Relay Hub contract has changed, and so failure would be desirable in order to bring this to attention